.dockerignore

```diff
@@ -0,0 +1,26 @@
+**/.classpath
+**/.dockerignore
+**/.env
+**/.git
+**/.gitignore
+**/.project
+**/.settings
+**/.toolstarget
+**/.vs
+**/.vscode
```

Comment:
While true that usually those should be excluded, in the context of CI you don't usually have any of the files excluded using the `.gitignore`, so I would probably not duplicate them.

That can cause bigger images when building it locally, but usually you don't want to do that either way, except for testing/debugging purposes